.travis.yml

```diff
@@ -3,6 +3,12 @@
 
 addons:
   postgresql: "9.4"
+  apt_packages:
+    - enchant
+    - aspell
+    - aspell-en
+    - ispell
+    - iamerican
```

Comment:
I would prefer that we do not enable sphinx-spelling if those packages are not found.

The basic pip install ./master[docs] should always work on any system.

We have ansible scripts to build the doc, which we would need to upgrade